lib/chromic_pdf/pdf/chrome_runner.ex

```diff
@@ -147,15 +147,37 @@ defmodule ChromicPDF.ChromeRunner do
   # https://github.com/bitcrowd/chromic_pdf/issues/76
   defp args(extra, opts) do
     default_args()
+    |> remove_conflicting_args(opts[:chrome_args])
     |> append_if("--no-sandbox", no_sandbox?(opts))
-    |> append_if(to_string(opts[:chrome_args]), !!opts[:chrome_args])
+    |> append_chrome_args_if(opts[:chrome_args])
```

Comment:
I think the implementation could use another round of polishing :)

```suggestion
    |> handle_chrome_args(opts[:chrome_args])
```

Maybe go for a unified handler function. 3 Clauses, `nil` if option not set, then a `is_binary` clause for the original behaviour, and a new `is_list` clause for handling the extended args. Extracting options from this list should then be something like:

```elixir
remove = List.wrap(Keyword.get(extended, :remove, []))
```